<launch>
	<arg name="ros_bridge_port"  default="9090" />

	<node name="teleop_key_ics" pkg="interactive_customer_service" type="teleop_key_ics" output="screen">
		<!--
		<remap from="/interactive_customer_service/message/customer" to="/interactive_customer_service/message/customer" />
		<remap from="/interactive_customer_service/message/robot"    to="/interactive_customer_service/message/robot" />
		<remap from="/interactive_customer_service/robot_status"     to="/interactive_customer_service/robot_status" />
		<remap from="/interactive_customer_service/customer_image"   to="/interactive_customer_service/customer_image" />
		-->
	</node>
	
	<node name="print_robot_status" pkg="rostopic" type="rostopic" output="screen" args="echo -p /interactive_customer_service/robot_status" />

<!--	
	<group ns="sigverse_ros_bridge">
		<node name="sigverse_ros_bridge" pkg="sigverse_ros_bridge" type="sigverse_ros_bridge" args="$(arg sigverse_ros_bridge_port) $(arg sync_time_num)"/>
	</group>
-->

	<node name="teleop_key_ics_input" pkg="interactive_customer_service" type="teleop_key_ics_input" output="screen" launch-prefix="gnome-terminal -t 'Teleop Key Input' -e">

		<param name="goods_list_path" value="$(find interactive_customer_service)/io/GoodsList.txt"/>

	</node>
	
	<include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch" > 
		<arg name="port" value="$(arg ros_bridge_port)"/>
	</include>
</launch>
